Kjt 20170508 adddescriptionstodetailspage

taylorkj opened the pull request.

srkirkland commented on AD419.DataHelper.Web/Views/Titles/Delete.cshtml:
```diff
@@ -104,7 +104,10 @@
-            @Html.DisplayFor(model => model.StaffType)
+            @{
+                var staffType = Model.StaffType;
+                @(string.IsNullOrWhiteSpace(staffType) ? Html.Raw("None") : Html.Raw(staffType));
```
@taylorkj where does staff type come from and why does it have to be raw?  This seems dangerous.

taylorkj commented:
@srkirkland : StaffType comes from the model.  I deleted the logic and added [DisplayFormat(NullDisplayText = "None")] to the model to produce the same results.
